package.json

```diff
@@ -108,6 +108,7 @@
     "slashify": "1.0.0",
     "sticky-sidebar": "3.3.1",
     "stream-mock": "2.0.5",
+    "testcafe-browser-provider-browserstack": "1.13.0",
```

Comment:
Maybe this should be added to `devDependencies` instead of `dependencies`. It's more for testing than to run the application so `dev` feels a better fit.